django/chat/static/chat/style.css

```diff
@@ -1271,3 +1271,28 @@ math[display="block"] {
   margin-top: 1rem;
   padding-bottom: 1rem;
 }
+
+.wand-button {
+  background-color: transparent;
```

Comment:
You can probably make this style rule work like this: `background-color: transparent !important;`